.github/CHANGELOG.md

```diff
@@ -380,36 +266,85 @@ K_test = qml.kernels.kernel_matrix(X_train, X_test, kernel)
-  + (0.25) [Y3 Y2 X1]
-  + (0.25) [Y3 X2 Y1]
-  + (0.25) [X4 X5 X2]
-  + (0.25) [Y4 Y5 X2]
-  + (0.25) [Y4 X5 Y2]
-  + (0.25) [X5 X4 X0]
-  + (0.25) [Y5 Y4 X0]
-  + (0.25) [Y5 X4 Y0]
-  >>> mapping
+  >>> print(mapping)
   {0: (0, 1), 1: (0, 2), 2: (1, 0), 3: (1, 2), 4: (2, 0), 5: (2, 1)}
   ```
- Additional functionality can be found in the `qml.qaoa.cycle` module.
+  Additional functionality can be found in the
+  [qml.qaoa.cycle](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.qaoa.cycle.html)
+  module.
+
+
+<h4>Extended operations and templates</h4>
+
+* Added functionality to compute the sparse matrix representation of a `qml.Hamiltonian` object.
+  [(#1394)](https://github.com/PennyLaneAI/pennylane/pull/1394)
+
+  Example:
+
+  ```python
+  coeffs = [1, -0.45]
+  obs = [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliY(0) @ qml.PauliZ(1)]
+  H = qml.Hamiltonian(coeffs, obs)
+  H_sparse = sparse_hamiltonian(H)
+  ```
+
+  The resulting matrix is a sparse matrix in scipy coordinate list (COO) format:
+
+  ```python
+  >>> H_sparse
+  <4x4 sparse matrix of type '<class 'numpy.complex128'>'
+      with 2 stored elements in COOrdinate format>
+  ```
+
+  The sparse matrix can be converted to an array as:
+
+  ```python
+  >>> H_sparse.toarray()
+  array([[ 1.+0.j  ,  0.+0.j  ,  0.+0.45j,  0.+0.j  ],
+         [ 0.+0.j  , -1.+0.j  ,  0.+0.j  ,  0.-0.45j],
+         [ 0.-0.45j,  0.+0.j  , -1.+0.j  ,  0.+0.j  ],
+         [ 0.+0.j  ,  0.+0.45j,  0.+0.j  ,  1.+0.j  ]])
+  ```
+
+* Adds the new template `AllSinglesDoubles` to prepare quantum states of molecules
+  using the `SingleExcitation` and `DoubleExcitation` operations.
+  The new template reduces significantly the number of operations
+  and the depth of the quantum circuit with respect to the traditional UCCSD
+  unitary.
+  [(#1383)](https://github.com/PennyLaneAI/pennylane/pull/1383)
+
+  For example, consider the case of two particles and four qubits.
+  First, we define the Hartree-Fock initial state and generate all
+  possible single and double excitations.
+
+  ```python
+  import pennylane as qml
+  from pennylane import numpy as np
+
+  electrons = 2
+  qubits = 4
+
+  hf_state = qml.qchem.hf_state(electrons, qubits)
+  singles, doubles = qml.qchem.excitations(electrons, qubits)
+  ```
+  Now we can use the template ``AllSinglesDoubles`` to define the
+  quantum circuit,
+
+  ```python
+  from pennylane.templates import AllSinglesDoubles
 
+  wires = range(qubits)
+
+  dev = qml.device('default.qubit', wires=wires)
+
+  @qml.qnode(dev)
+  def circuit(weights, hf_state, singles, doubles):
+      AllSinglesDoubles(weights, wires, hf_state, singles, doubles)
+      return qml.expval(qml.PauliZ(0))
+
+  params = np.random.normal(0, np.pi, len(singles) + len(doubles))
+  circuit(params, hf_state, singles=singles, doubles=doubles)
+  ```
 * Adds `QubitCarry` and `QubitSum` operations for basic arithmetic.
   [(#1169)](https://github.com/PennyLaneAI/pennylane/pull/1169)
```

Comment:
I can't comment on it, but same comment before - the split between python/pycon can take place when `circuit()` is evaluated 🙂